scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -430,9 +730,6 @@ BA_ "GenMsgCycleTime" BO_ 508 1000;
 BA_ "GenMsgCycleTime" BO_ 509 1000;
 BA_ "GenMsgCycleTime" BO_ 510 1000;
 
-BA_ "GenSigStartValue" SG_ 2  tx_overflow_count 0;
```

Comment:
@Marko/Prom do these 2 lines do anything?

Reply:
This may be checking for overflow in can tx/rx mailboxes. Were these accidentally removed?

Reply:
There is no _BO_ 2_ message ID in the original file so I don't think they were accidentally removed.